src/styles/less/NextSteps.less

```diff
@@ -86,13 +86,41 @@
 
 .overlay-image {
     position: absolute;
-    top: 0;
+    top: 10%;
     left: 0;
     margin: -3rem 0rem 0rem 3rem;
+
+    @media (max-width: 560px) {
+        transform: scale(90%);
+        left: -15%;
+        top: 0;
+    }
+
+    @media (max-width: 380px) {
+        transform: scale(80%);
+        left: -5rem;
+        top: 0;
+    }
 }
```

Comment:
Great job including the additional breakpoints for smaller screens. The before and after images are very helpful to visualize the differences for anyone who cannot readily test the code on their local machine!